foundation/scheduler/runnable.go

```diff
@@ -12,12 +12,12 @@ const (
 	ChangeTypeStop  ChangeEvent = iota
 )
 
-// Runnable describes something that is runnable
-type Runnable interface {
-	// Run is the entrypoint for jobs handled by a Runnable
+// Module describes a Wasm module
```

Comment:
Correct me if I'm wrong but the scheduler is still capable of executing multiple types of runnables of which wasm *modules* happen to be one.